code/datums/ghost_query.dm

```diff
@@ -20,10 +20,10 @@
 
 	// Then wait awhile.
 	while(!finished)
-		sleep(1 SECOND)
-		wait_time -= 1 SECOND
-		if(wait_time <= 0)
-			finished = TRUE
+		spawn(1 SECOND)
```

Comment:
revert this

Reply:
This was replaced with spawn because linter refuses to accept any form of sleeping in initialization.

Reply:
well then the linter needs a smack, 'cause polaris' version of this code still uses sleep

Reply:
Because Polaris I don't believe has any ghostpods that activate instantly upon spawning. We do. Hence the difference.